### ISSUE TITLE
Cannot find module app/dist/main.js

### ISSUE THREAD

**Issue by ulidtko** [ewnd9/record-desktop#28]
The 0.5.2-1 .deb from github/releases doesn't work:

```
% record-desktop
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/usr/share/record-desktop/resources/app/dist/main.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Object.<anonymous> (/usr/share/record-desktop/resources/atom.asar/browser/init.js:158:8)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:151:18)
    at node.js:1007:3
^C%

% dpkg -L record-desktop | grep main.js
/usr/share/record-desktop/resources/app/frontend/main.js
/usr/share/record-desktop/resources/app/node_modules/mksnapshot/lib/main.js
/usr/share/record-desktop/resources/app/node_modules/resolve/test/module_dir/zmodules/bbb/main.js
/usr/share/record-desktop/resources/app/node_modules/resolve/test/pathfilter/deep_ref/main.js
/usr/share/record-desktop/resources/app/node_modules/resolve/test/precedence/aaa/main.js
/usr/share/record-desktop/resources/app/node_modules/resolve/test/precedence/bbb/main.js
/usr/share/record-desktop/resources/app/node_modules/resolve/test/resolver/without_basedir/main.js
/usr/share/record-desktop/resources/app/node_modules/tough-cookie/lib/permuteDomain.js
/usr/share/record-desktop/resources/app/src/main.js
```

I think the deb packaging is slightly off with regards to install paths.

**Comment by ewnd9** [comment]
Thanks for pointing out. Unfortunately, the two latest versions are broken (don't contain transpiled sources). I've tried to apply a quick fix and v0.5.5 is starting, but there is another error in frontend part due to packaging. 

I'm going to deal with it on the weekend.

**Comment by ulidtko** [comment]
@ewnd9 thanks for your care. Hope you'll sort it out!

**Comment by ewnd9** [comment]
I've published [`v0.5.6`](https://github.com/ewnd9/record-desktop/releases/tag/v0.5.6) which works for me on my laptop and in VirtualBox (both Ubuntu 16.04).

Hope you will give it another chance :v:

**Comment by ewnd9** [comment]
Feel free to reopen if the problem is still there.
